### DISCUSSION TITLE
Remember user list border location

### DISCUSSION THREAD

**Discussion by BeerNecessities** [nicotine-plus/nicotine-plus#2633] [Ideas]
It would be nice if Nicotine+ remembered the user list border location when you reopen Nicotine+.  Also for the app to remember window location when opened again.  Thanks everyone to all the work you do and for a wonderful client app!

**Comment by mathiascode** [comment]
Do you mean the chat room user list, or the buddy list?

Regarding window position, what operating system and version of Nicotine+ are you using? If you're using the unstable build of Nicotine+, GTK 4 removed the API to set the window location. It would probably be possible to interact with operating system APIs directly on Windows, macOS and X11, though. If you're using the stable build, I'm not sure what's wrong, since the window position should already be set.

**Reply by BeerNecessities** [reply]
Hi.  I meant chat room user list.  I am using the latest stable build 3.2.9  •  GTK 3.24.36.

**Reply by mathiascode** [reply]
In what way are you resizing the pane, to show more or less? I'm thinking about making it resize automatically depending on the window size, so there's enough room for the columns and it doesn't end up in weird states.

**Reply by BeerNecessities** [reply]
Hi, I have the app in the bottom half of my screen on portrait mode.  When I open the app in chat rooms I minimize (less) the user list window but when I close the app it forgets the position of that border when I reopen the app.
